support/k8s/k8s-workload-registrar/generate-config.go

```diff
@@ -88,7 +88,7 @@ users:
 
 # Validating Webhook Configuration for the K8S Workload Registrar
 #
-apiVersion: admissionregistration.k8s.io/v1beta1
+apiVersion: admissionregistration.k8s.io/v1
```

Comment:
Were you able to verify config outputted here works? I thought the v1 webhook requires some additional fields like `sideEffects`, `failurePolicy`, etc.

Reply:
again good catch @faisal-memon, we looked at the output of the `kubectl explain` command and realized that the following anecdotes:

```shell
 failurePolicy        <string>
     FailurePolicy defines how unrecognized errors from the admission endpoint
     are handled - allowed values are Ignore or Fail. Defaults to Fail.

 admissionReviewVersions      <[]string> -required-
     AdmissionReviewVersions is an ordered list of preferred `AdmissionReview`
     versions the Webhook expects. API server will try to use first version in
     the list which it supports. If none of the versions specified in this list
     supported by API server, validation will fail for this object. If a
     persisted webhook configuration specifies allowed versions and does not
     include any versions known to the API Server, calls to the webhook will
     fail and be subject to the failure policy.

 sideEffects  <string> -required-
     SideEffects states whether this webhook has side effects. Acceptable values
     are: None, NoneOnDryRun (webhooks created via v1beta1 may also specify Some
     or Unknown). Webhooks with side effects MUST implement a reconciliation
     system, since a request may be rejected by a future step in the admission
     chain and the side effects therefore need to be undone. Requests with the
     dryRun attribute will be auto-rejected if they match a webhook with
     sideEffects == Unknown or Some.

 timeoutSeconds       <integer>
     TimeoutSeconds specifies the timeout for this webhook. After the timeout
     passes, the webhook call will be ignored or the API call will fail based on
     the failure policy. The timeout value must be between 1 and 30 seconds.
     Default to 10 seconds.
```

based on the anecdotes above, we should only add these two fields: `admissionReviewVersions` and `sideEffects`.

Reply:
Yes, you're right.